sources/packages/web/src/services/BypassedRestrictionService.ts

```diff
@@ -0,0 +1,27 @@
+import ApiClient from "@/services/http/ApiClient";
+import { BypassedRestrictionSummaryAPIOutDTO } from "@/services/http/dto";
+
+/**
+ * Client service layer for Bypassed Restrictions.
+ */
+export class BypassedRestrictionService {
+  // Shared Instance
+  private static instance: BypassedRestrictionService;
+
+  static get shared(): BypassedRestrictionService {
+    return this.instance || (this.instance = new this());
+  }
+
+  /**
+   * Get history of bypassed restrictions for an application that is not a draft.
+   * @param applicationId application id.
+   * @returns summary of the bypassed restriction history for a student application.
+   */
+  async getBypassedRestrictionHistory(
```

Comment:
I would suggest the name of Service and Api method must be more generic and can be the same as API service method `getApplicationRestrictionBypasses`

Reply:
Updated.